src/main/java/com/jcabi/w3c/DefaultValidationResponse.java

```diff
@@ -131,12 +129,18 @@ public Charset charset() {
 
     @Override
     public Set<Defect> errors() {
-        return Collections.unmodifiableSet(this.ierrors);
+        synchronized (this.ierrors) {
```

Comment:
@piddubnyi if we use CopyOnWriteArraySet we don't need to use synchronized block and to copy the collection before return it